prow/spyglass/lenses/links/links.go

```diff
@@ -36,6 +37,8 @@ const (
 	priority = 20
 
 	bytesLimit = 10 * 1024
+
+	defaultGCSBrowserPrefix = "https://gcsweb.k8s.io/gcs/"
```

Comment:
There is config.spyglass.gcs_browser_prefix, please use that instead of  a hardcoded value

Reply:
I wanted to use that value initially, but it looks like lens doesn't have access to spyglass config (only to lens config)

Reply:
So lets plug it through?

Reply:
I opened https://github.com/kubernetes/test-infra/pull/21373. Please review this change :)

Reply:
Done